nautobot/circuits/models.py

```diff
@@ -310,22 +314,7 @@ def __str__(self):
         return f"Termination {self.term_side}: {self.site or self.provider_network}"
 
     def get_absolute_url(self):
-
-        # Circuit terminations can terminate on a site or a provider network.
-        # There is no separate view for Circuit Terminations,
-        # so we return the absolute URL of the corresponding terminal here.
-        if self.site:
-            return self.site.get_absolute_url()
-        return self.provider_network.get_absolute_url()
-
-    def clean(self):
-        super().clean()
-
-        # Must define either site *or* provider network
-        if self.site is None and self.provider_network is None:
-            raise ValidationError("A circuit termination must attach to either a site or a provider network.")
-        if self.site and self.provider_network:
-            raise ValidationError("A circuit termination cannot attach to both a site and a provider network.")
```

Comment:
This looks like an unintended set of changes, possibly resulting from moving from `develop` to `next` - can you please check this?

Reply:
Yep, this was unintended, I think it was because I changed the function above it, and then when I repointed branches, git decided I wanted to delete this chunk as well...